components/docs-chef-io/content/automate/restore.md

```diff
@@ -52,6 +52,13 @@ Ensure access for the backup type used:
          # """
     ```
 
+1. To restore a backup to a machine with less memory than the original system, adjust for the appropriate lower memory settings by adding the following to the configuration file:
+
+    ```toml
+    [elasticsearch.v1.sys.runtime]
+      heapsize = 4096m
```

Comment:
I think you need to specify the units for the heapsize.

Reply:
@mattray Do you have more information on heapsize units used?

Reply:
@mjingle @mattray O sorry -- I should have looked more closely. The `m` is showing up in red for me both in the diff view, and I thought that meant it had been deleted. It's also showing up red in the [full file view](https://github.com/chef/automate/pull/4270/files?short_path=27930b7#diff-27930b7e654e98bda569c8f108979683) so I'm not sure what's going on.

Reply:
No worries. Usually if a character shows up with a red text background, it is caused by the error-highlighting feature of GitHub's text editor/viewer.

Reply:
@mjingle Aha -- I think it needs to be `heapsize ="4096m"` with double quotes.

Reply:
@yzl 😄  Great eye! Thanks! I'll make that change also. 

@mattray If you would like, we can also include info on heapsize units. :)

Reply:
I'm having a hard time tracking down an authoritative source but it's all over the elasticsearch and JVM documentation. Putting "4096m" will be megabytes and "4g" will be gigabytes.

Reply:
@mattray Understood. Thank you!